frontend/app/views/fragments/form/stripeCheckout.scala.html

```diff
@@ -0,0 +1,13 @@
+@import views.support.IdentityUser
+@(idUser: IdentityUser)
+<script>
+    guardian.stripeCheckout = {
+        key: guardian.stripePublicKey,
+        image: 'https://d24w1tjgih0o9s.cloudfront.net/gu.png',
+        locale: 'auto',
+        name: 'Guardian',
+        zipCode: false,
+        allowRememberMe: false
+    };
+</script>
+<button type="button" class="action choose-payment-method js-stripe-checkout" data-email="@idUser.email">stripe checkout, yeah</button>
```

Comment:
Hey @AWare, can you explain what the `data-email` field is used for? What breaks if I take it away?